### PR TITLE
re-joining codes to old P5.js Tutorials and sorting folders

### PR DESCRIPTION
**Pull request by chepecarlos**
A few months ago I took the Tutorial "# 8: HTML / CSS / DOM", it helped me a lot, but I could not find the code, I managed to find them in github because I know a little bit of git.

A month ago I used the code from thecodingtrain.com/ to create my own page http://nocheprogramacion.com/ I understand a little more the system, I see that many repository is not correct. I suppose that when there was still a moment in the step they worked but they do not do it anymore.

I thought that another user would like to learn and I could not find the code, I started to order Tutorial # 8, but I see what happened in the others. I have ordered all the codes to appear on the page and with the folder structure that you are using in _Tutorial

I will then try to add on the basis of the videos those that are not yet documented in the repository

Thank you for the great work, I love your video and the work of the community, sorry for a lot of code, I hope that someone else will be useful

## Example before
Antes
## Example after
Despues

### CODE REVIEW

**Comment by chepecarlos** [comment]
This is what I did:
* Each code of P5.js moves to its own folder P5
* If there are multiple codes for a tutorial everything is put in folder P5 with its original name
* Sorting and changing name folders to be more similar _Tutorials
* Deleting library that I found a code and the reference of index.html that were left by mistake, library p5.serialport.js
* Adding the option of video to deactivate live example video by video 
* Deactivating the live_example until creating live
* New code address of folder P5

File: _Tutorials/1-p5js-basics/1.3-basics-of-drawing.md [hunk 1]
```diff
@@ -3,10 +3,12 @@ title: Basics of Drawing
...
+repository: P5JS/01-p5js-basics/1.3_p5.js_basics_of_drawing
+live_example: false
 
 links:
   - title: Reference
     url: https://p5js.org/reference
 ---
 
-In this video, I teach you about the basics of drawing shapes in a web browser using the p5.js library.
+In this video 2ASSS, I teach you about the basics of drawing shapes in a web browser using the p5.js library.
```
Comment (gruselhaus) [inline _Tutorials/1-p5js-basics/1.3-basics-of-drawing.md]: Is this intended?

File: _Tutorials/7-object-interaction/7.5-removing-objects-from-array.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ title: "Removing Objects from Arrays"
 video_number: 5
 date: 2017-10-17
 video_id: tA_ZgruFF9k
-repository: /Tutorials/P5JS/p5.js/07/7.5_1_p5js_Removing_Objects_from_Arrays
```
Comment (gruselhaus) [inline _Tutorials/7-object-interaction/7.5-removing-objects-from-array.md]: This should always be: `/Tutorials/...`.

**Comment by nilsweb** [comment]
Thanks a lot for your help, @chepecarlos! 🎉 
I've taken a quick look at your changes and from what I've seen so far, everything should work as intended. 👍 I'll have a closer look over the next week to see, if there are any problems.

**Comment by gruselhaus** [comment]
Closing this because this pull request fundamental merge conflicts.

**Pull request closed without merging**